<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomProject</class>
 <widget class="QWidget" name="CustomProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project &amp;Management</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Default open &amp;project directory :</string>
        </property>
        <property name="buddy">
         <cstring>m_projectPathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DirectoryEditWidget" name="m_projectPathLineEdit" native="true">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="whatsThis">
         <string>&lt;p&gt;This the path where the save dialog must be opened when you create a new project.&lt;/p&gt;
&lt;p&gt;&lt;i&gt;As the author set the project path to the Application Server path, this path is now the project path. You can safely use another path, but in &lt;i&gt;New Project Wizard&lt;/i&gt;, the Application Server path would be wrong.&lt;/i&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Refresh Directory &amp;Timeout :</string>
        </property>
        <property name="buddy">
         <cstring>m_refreshProjectTimoutSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_refreshProjectTimoutSpinBox">
        <property name="whatsThis">
         <string>Timeout before the directory view is reloaded with your filter.</string>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="m_openLastProjectCheckBox">
        <property name="whatsThis">
         <string>Open the last opened project (&lt;i&gt;if any, it's the opened project when &lt;b&gt;XINX&lt;/b&gt; is closed&lt;/i&gt;), when &lt;b&gt;XINX&lt;/b&gt; is started.</string>
        </property>
        <property name="text">
         <string>&amp;Open the last project at start</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="m_saveSessionCheckBox">
        <property name="whatsThis">
         <string>&lt;p&gt;This define the default way to close the project.&lt;/p&gt;
&lt;p&gt;If checked, when the project, or the application is closed informations about unsaved files are stored in a session file.&lt;/p&gt;
&lt;p&gt;Else, the XINX ask the user about unsaved file.&lt;/p&gt;
&lt;p/&gt;
&lt;p&gt;&lt;i&gt;In case of crash &lt;b&gt;XINX&lt;/b&gt; save the project in session mode automatically.&lt;/i&gt;&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Save project with session information by default (content of modified file)</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>m_refreshProjectTimoutSpinBox</zorder>
     <zorder>label_20</zorder>
     <zorder>m_projectPathLineEdit</zorder>
     <zorder>label_2</zorder>
     <zorder>m_openLastProjectCheckBox</zorder>
     <zorder>m_saveSessionCheckBox</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>General &amp;Revision Control System Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="m_closeLogCheckBox">
        <property name="whatsThis">
         <string>&lt;p&gt;When an action with the revision control system is made, a log window is open and show information about the progress.&lt;/p&gt;
&lt;p&gt;If checked, the log windows is automaticallyclosed when action (like update, commit ...) finished.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Close the Version management Log automatically.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="m_changeLogCheckBox">
        <property name="whatsThis">
         <string>&lt;p&gt;When checked, a &lt;i&gt;ChangeLog&lt;/i&gt; file is created/appended at each commit with date, hour, and the commit messages.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Update &quot;ChangeLog&quot; automatically.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="m_autoAddFileChk">
        <property name="text">
         <string>&amp;Auto add file to versionning system</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEditWidget</class>
   <extends>QWidget</extends>
   <header>directoryedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
